### DISCUSSION TITLE
Lost private key

### DISCUSSION THREAD

**Discussion by MUTOYASAR** [IBAX-io/go-ibax#1049] [Q&A] [answered]
create or import account --- generate new key---- you have generate word and password confirm 
this way you can login to your wallet

**Comment by SamanCoding** [comment, marked as answer]
The password is used to encrypt your private key and allow easy access to your account.

IBAX does not store your private key, so the private key associated with the password will be stored in your browser's cookie.

When you clear the cookie or use a new browser to access your IBAX account, you will need to import your helper notes and encrypt them again with your password.

Of course you can also regenerate an IBAX account and re-encrypt it with a password

**hint：**

- Take care of your private key (account seed)
- Take care of your public key
- Take care of your password
- Take care of your wallet address

Please refer to this discussion:  #62
